mbrl/models/model_trainer.py

```diff
@@ -71,6 +72,8 @@ def train(
         patience: Optional[int] = None,
         improvement_threshold: float = 0.01,
         callback: Optional[Callable] = None,
+        meta_includes_grad_norm: bool = False,
```

Comment:
Hmm, it seems like this arg is getting passed around a lot. Would it make sense to create a set of default post-loss callbacks and include this one in it? That would prevent the number of arguments from ballooning

Reply:
That is, this seems like a subset of meta-data you might want a model to return but this arg seems to be migrating out into a lot of classes in a way that might make refactoring more difficult in the future

Reply:
Right. And also the main purpose of this is to avoid computing this data if it's not really needed, since you have to aggregate over all model parameters. But, I do agree with your general comment about the argument being passed around a lot. 

On the other hand, very likely the cost of logging this is small compared to forward/backward, so maybe this is a case of over-optimization? We might just get rid of this arg altogether and just include grad norm every time? Or maybe turn it off via a model method, rather than via the model trainer.

Reply:
Yeah, agree that I'm probably over-optimizing here. I think it probably makes since to always include it since it for standard tiny RL models the cost of computing will be tiny and cut down on extra args

Reply:
Actually, I meant I was over-optimizing by including this arg to avoid computing the grad. I think this is a good suggestion, so I'll just remove the arg to keep the code cleaner, and consider some way of turning it off if it ever becomes an issue later.

Reply:
Done.